packages/proton-bridge/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ProtonMail/proton-bridge
+TERMUX_PKG_DESCRIPTION="ProtonMail Bridge application"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=1.4.5
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+
+termux_step_get_source() {
+	termux_setup_golang
+	export GOPATH=$TERMUX_PKG_BUILDDIR
```

Comment:
When termux_step_get_source is no longer overridden these lines can be moved to the beginning of termux_step_make_install.

Reply:
Seems to work ok!